### PR TITLE
doc(routes): add sub-title

### PR DESCRIPTION
**Pull request by ahuigo**
For: https://github.com/denoland/fresh/issues/547

### CODE REVIEW

**Comment by excessivecoding** [comment]
Related issue & answer from maintainer: https://github.com/denoland/fresh/issues/491#issuecomment-1181769820

**Comment by ahuigo** [comment]
> Related issue & answer from maintainer: [#491 (comment)](https://github.com/denoland/fresh/issues/491#issuecomment-1181769820)

As it is redundant, I think I need to delete the description about default layout(`_app.tsx`),  and keep the sub-title only.

If there is any inappropriateness, feel free to close the PR.

**Review by lucacasonato — APPROVED** [review]
Thanks

File: docs/concepts/routes.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ description: |
   Routes are the basic building block of fresh applications. They are used to define the behaviour the application when a given path is requested.
 ---
 
+## Preface
+
```
Comment (lucacasonato) [inline docs/concepts/routes.md]:
```suggestion
```

**Pull request merged**